### ISSUE TITLE
[ER] option /command to allow camera to use the front camera 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#174]
```
When a front camera exist add an option / command to take a picture with the 
front Camera


```

Original issue reported on code.google.com by `lpenou.tstXX@mpo.li` on 20 May 2011 at 7:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 21 May 2011 at 6:51
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
camera:list to show all available cameras.
camera:set:X to choose the camera (0 by default)
```

Original comment by `Florent....@gmail.com` on 10 Jun 2011 at 8:13
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.2.4
```

Original comment by `Florent....@gmail.com` on 13 Jun 2011 at 1:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
